api/src/main/java/org/openmrs/Obs.java

```diff
@@ -106,49 +132,64 @@
 	 * 
 	 * @see #isObsGrouping() (??)
 	 */
+	@JoinColumn(name = "obs_group_id")
+	@ManyToOne
 	protected Obs obsGroup;
 	
 	/**
 	 * The list of obs grouped under this obs.
 	 */
 	@AllowDirectAccess
+	@JoinColumn(name = "obs_group_id")
+	@OneToMany(cascade = CascadeType.REMOVE)
+	@OrderBy("obs_id")
+	@BatchSize(size = 25)
 	protected Set<Obs> groupMembers;
-	
+	@JoinColumn(name = "value_coded")
+	@ManyToOne
 	protected Concept valueCoded;
-	
+	@JoinColumn(name = "value_coded_name_id")
+	@ManyToOne
 	protected ConceptName valueCodedName;
-	
+	@JoinColumn(name = "value_drug")
+	@ManyToOne
 	protected Drug valueDrug;
-	
+	@Column(name = "value_group_id", length = 11)
 	protected Integer valueGroupId;
-	
+	@Column(name = "value_datetime", length = 19)
 	protected Date valueDatetime;
-	
+	@Column(name = "value_numeric", length = 22)
 	protected Double valueNumeric;
-	
+	@Column(name = "value_modifier", length = 2)
 	protected String valueModifier;
-	
+	@Column(name = "value_text", length = 65535)
 	protected String valueText;
-	
+	@Column(name = "value_complex", length = 255)
 	protected String valueComplex;
 	
 	// ComplexData is not persisted in the database.
 	protected transient ComplexData complexData;
-	
+	@Column(name = "comments", length = 255)
 	protected String comment;
-	
+	@Transient
```

Comment:
Is this really transient? Try to match the original configuration in the xml file

Reply:
It was an oversight...It is not transient but rather not insertable.....its generated by the db itself